### PR TITLE
Extension removed from URI and query variables change to post

### PR DESCRIPTION
**Pull request by nunogit**

### CODE REVIEW

File: demo/components/http-endpoint/http-endpoint-service.js [hunk 1]
```diff
@@ -41,16 +41,18 @@ app.service('HttpEndpoint', function($q, $http, $timeout, GENERAL_CONFIG) {
     load : function(resource, graphUri) {
       var deferred = $q.defer();
       var cacheLocation = endpoint;
-      if (graphUri) {
-        cacheLocation = cacheLocation + '?uri='+resource+'&context-uri=' + graphUri;
-      } else {
-        cacheLocation = cacheLocation + '?uri='+resource+'&context-uri=' + resource;
+      var context-uri = resource
```
Comment (rajaram5) [inline demo/components/http-endpoint/http-endpoint-service.js]: Shall we use only camelCase!

File: demo/components/http-endpoint/http-endpoint-service.js [hunk 2]
```diff
@@ -41,16 +41,18 @@ app.service('HttpEndpoint', function($q, $http, $timeout, GENERAL_CONFIG) {
...
       }
       
+	  	
+	
       $http.post(cacheLocation, {
         headers : {
           'Content-Type' : 'application/x-www-form-urlencoded'
-        }
+        },
+		data: { uri: resource, context-uri: graphUri }
```
Comment (rajaram5) [inline demo/components/http-endpoint/http-endpoint-service.js]: Shouldn't this line start at same level as "headers" line?

**Review by rajaram5 — CHANGES_REQUESTED** [review]
Just some minor comments about the indentation and the variable naming. BTW, @nunogit I know that in my previous commits to this project I didn't follow the code formatting :-) could you be kind enough to format these two files so that these files have proper indentation!!

File: demo/components/http-endpoint/http-endpoint-service.js (hunk 1 above)
Comment (nunogit) [reply demo/components/http-endpoint/http-endpoint-service.js]: :) np, I will

File: demo/components/http-endpoint/http-endpoint-service.js (hunk 2 above)
Comment (nunogit) [reply demo/components/http-endpoint/http-endpoint-service.js]: yes. That's a weird bug.

Comment (nunogit) [reply demo/components/http-endpoint/http-endpoint-service.js]: True, maybe some weird last minute typo

Comment (nunogit) [reply demo/components/http-endpoint/http-endpoint-service.js]: I missunderstood what you said. What do you mean?

Comment (rajaram5) [reply demo/components/http-endpoint/http-endpoint-service.js]: I mean the indentation in line 55 is wrong. Both the lines 52 and 55 should have same indentation right?

Comment (nunogit) [reply demo/components/http-endpoint/http-endpoint-service.js]: Ah, yes.. I also fixed that.

**Review by rajaram5 — APPROVED** [review]
Looks good to me, we still have to discuss about the indentation but you can merge this changes

**Comment by kburger** [comment]
As Rajaram said, the indentation is a big issue. It shouldn't be hard to fix that with a proper IDE or editor.

File: demo/components/http-endpoint/http-endpoint-service.js [hunk 3]
```diff
@@ -1,55 +1,59 @@
...
-      }).then(function(res) {
-        deferred.resolve(res);
-        console.log('Caching : url ==>  ' + cacheLocation);
-      }, function(res) {
-        console.log('failed to PUT file in endpoint');
-        deferred.reject(res);
+	headers: {
+	  'Content-Type': 'application/x-www-form-urlencoded'
+	},
+	data: {
```
Comment (rajaram5) [inline demo/components/http-endpoint/http-endpoint-service.js:47]: @nunogit the POST call is not working. I get error when I try to cache fdp

**Pull request merged**